configmap/repl-wrk/start/start.sh

```diff
@@ -69,7 +69,7 @@ QSERV_WORKER_DB_URL="mysql://${QSERV_WORKER_DB_USER}:${QSERV_WORKER_DB_PASSWORD}
 entrypoint --log-level DEBUG smig-update --worker-connection "mysql://${QSERV_WORKER_DB_USER}:${QSERV_WORKER_DB_PASSWORD}@${QSERV_WORKER_DB_DN}:${QSERV_WORKER_DB_PORT}"
 
 
-qserv-replica-worker ${WORKER_ID} --config=${CONFIG} --qserv-worker-db="${QSERV_WORKER_DB_URL}" --debug
```

Comment:
@iagaponenko could you please remove $WORKER_ID definition on top of the script and then merge?